apps/src/templates/SchoolAutocompleteDropdown.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
+
+export default class SchoolAutocompleteDropdown extends Component {
+
+  state = {};
+
+  render() {
+    const getOptions = (q) => {
+      if (!q.length) {
+        return Promise.resolve({ options: [{ value: -1, label: "School not found"}] });
+      }
+      return fetch('/dashboardapi/v1/schoolsearch/' + encodeURIComponent(q) + '/40')
+        .then((response) => response.json())
+        .then((json) => {
+          var schools = [];
+          for (var i = 0; i < json.length; i++) {
+            schools.push({
+              value: json[i].nces_id,
+              label: json[i].name + ' - ' + json[i].city + ', ' + json[i].state + ' ' + json[i].zip
+            });
+          }
+          if (schools.length > 1) {
+            return { options: schools };
+          } else {
+            return { options: [{ value: -1, label: "School not found"}] };
+          }
+      });
+    };
+
+    return (
+      <div>
+        <VirtualizedSelect
+          id="nces_school"
+          clearable={true}
+          async={true}
+          loadOptions={getOptions}
+          filterOption={() => true}
+          value={this.state.selectValue}
+          disabled={false}
+          onChange={(selectValue) => this.setState({ selectValue })}
```

Comment:
Do we consume the selected value of this dropdown anywhere?

Reply:
I use selectedValue to check if we need to show the SchoolNotFound input fields. I also need a way for whichever form is incorporating this component to know the value of this field. In this case, how do I pass the selectedValue as data to `census.rb`?

Reply:
I'm not super familiar with Pegasus forms, but it looks like it may be an ordinary form submission so as long as the dropdown is on a form and has a name it should be sent.

Reply:
Actually, I need the parent (in this case, `CensusForm`) to know the selected value so it can validate that the field isn't blank on submission.  Is there a way to access `SchoolAutocomplete`'s state from `CensusForm`? Or do I need to change the onChange function to be a callback up to the parent? But, in the future we want to use this dropdown in places where it won't have a React parent.